examples/player/simid_player.js

```diff
@@ -131,6 +131,30 @@ class SimidPlayer {
     });
   }
 
+  /**
+   * Player closes the ad and SIMID creative.
+   */
+  playerCloseAd() {
+    this.stopAd();
+    this.sendLog("Ad stopped by player");
```

Comment:
I think the logs in these functions are redundant.  The stop ad message contains a stop code saying that it is a player initiated stop. Does sending the log provide any additional value?

Reply:
good for sample code to show the user the message sent from player